plugins/timelib/timelib.cpp

```diff
@@ -382,13 +382,13 @@ time_t tlMKTime(struct tm* timeInfoPtr, bool inLocalTimeZone)
         the_time += (tlLocalTimeZoneDiffIn100nsIntervals() / _onesec_in100ns);
     }
     #else
```

Comment:
Should there be an equivalent change in tlWinMKTime?  I think it will work anyway since that function uses GMT.

Reply:
It does the right thing already.  However, there is an efficiency that I had and took out again for minimalistic reasons:  It always converts from UTC to local time.  If the original request was for UTC, there is another conversion.  The efficiency would be to pass is_local_time to that function and convert either zero or one time.